plugins/LilyPad/LilyPad.rc

```diff
@@ -274,6 +274,70 @@ BEGIN
     PUSHBUTTON      "Small Motor",ID_SMALL_MOTOR,347,291,64,14
 END
 
+IDD_CONFIG_PS1_MOUSE DIALOGEX 0, 0, 424, 318
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION
+FONT 8, "MS Shell Dlg", 0, 0, 0x1
+BEGIN
+    CONTROL         "",IDC_LIST,"SysListView32",LVS_REPORT | LVS_SHOWSELALWAYS | LVS_NOSORTHEADER | WS_BORDER | WS_VSCROLL | WS_TABSTOP,7,7,183,285,WS_EX_CLIENTEDGE
+    PUSHBUTTON      "Delete Selected",ID_DELETE,7,296,59,15
+    PUSHBUTTON      "Clear All",ID_CLEAR,71,296,56,15
+    PUSHBUTTON      "Ignore Key",ID_IGNORE,132,296,58,15
+    PUSHBUTTON      "Mouse",ID_MOUSE,366,30,34,15
+    PUSHBUTTON      "X-axis Left",ID_LSTICK_LEFT,256,112,45,15
+    PUSHBUTTON      "X-axis Right",ID_LSTICK_RIGHT,307,112,45,15
+    PUSHBUTTON      "Y-axis Up",ID_LSTICK_UP,283,92,45,15
+    PUSHBUTTON      "Y-axis Down",ID_LSTICK_DOWN,283,132,45,15
+    PUSHBUTTON      "Left-click", ID_CIRCLE,256,56,45,15
+    PUSHBUTTON      "Right-click", ID_CROSS,307,56,45,15
```

Comment:
Spaces after the comma here and the line above probably aren't meant to be there.

Apart from that, code looks ok, but I'm booted into Linux right now so I don't know what the dialog looks like (can't use my Windows VM either because I'm switching distros and haven't got things fully set up).